pontoon/base/static/css/style.css

```diff
@@ -952,6 +966,27 @@ header nav .right {
   float: right;
 }
 
+header nav .right .sign-in-header {
+  display: inline-block;
+  vertical-align: bottom;
+  font-size: 14px;
+  margin-bottom: 14px;
```

Comment:
13px (for proper vertical center-alignment).

Reply:
Are you telling me about the margin-bottom? Because I tried that and it was not centrally aligned in the header.